client/modules/accounts/components/auth/loginButtons.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component, PropTypes } from "react";
+import {
+  Button,
```

Comment:
I'm not a fan of doing that unless the line becomes too long. Otherwise it hurts my eyes.